### ISSUE TITLE
Add @OnlyOnServer Annotation for Properties

### ISSUE THREAD

**Issue by hendrikebbers** [canoo/dolphin-platform#88]
The annotation can be added to a property. By doing so the property won't be synchronized with the client.

**Comment by netopyr** [comment]
I do not think that this is a good idea. DolphinBeans are DTOs to synchronize data between client and server. If we make them more powerful, their responsibility will become blurred.

**Comment by hendrikebbers** [comment]
In that case we need to rethink the i18n implementation. I think it don't make sense to sync the key to the client. But to be true I currently don't see an additional point why we need it. So maybe rethink i18n :)
